### PR TITLE
Update the game over message.

### PR DESCRIPTION
**Pull request by IvanLopez-IDS**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by IvanLopez-IDS — COMMENTED** [review]
task pull

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
```
Comment (IvanLopez-IDS) [inline index.html:49]: comment

File: index.html [hunk 2]
```diff
@@ -148,5 +148,5 @@ <h2 hidden>Game over</h2>
       array[j] = temp;
     }
     return array;
-  }
+  } 
 </script>
```
Comment (Ivan-Lopez1) [inline index.html:152]:
```suggestion
</script>
```

File: index.html [hunk 3]
```diff
@@ -148,5 +148,5 @@ <h2 hidden>Game over</h2>
       array[j] = temp;
     }
     return array;
-  }
+  }
```
Comment (IvanLopez-IDS) [inline index.html]:
```suggestion
  } task pull
```

**Pull request merged**